src/Bindings/PluginManager.h

```diff
@@ -119,6 +119,7 @@ class cPluginManager
 		HOOK_PLAYER_PLACING_BLOCK,
 		HOOK_PLAYER_RIGHT_CLICK,
 		HOOK_PLAYER_RIGHT_CLICKING_ENTITY,
+		HOOK_PLAYER_CLICKING_AT_ENTITY,
```

Comment:
We've got "right-clicking" and "clicking", how about making this "left-clicking" so that the naming convention is the same?

Reply:
See "Is this right-click, left-click or any-click?"